src/Mod/Spreadsheet/Gui/Command.cpp

```diff
@@ -336,8 +336,8 @@ CmdSpreadsheetAlignCenter::CmdSpreadsheetAlignCenter()
 {
     sAppModule = "Spreadsheet";
     sGroup = QT_TR_NOOP("Spreadsheet");
-    sMenuText = QT_TR_NOOP("Align &center");
-    sToolTipText = QT_TR_NOOP("Center-align contents of selected cells");
+    sMenuText = QT_TR_NOOP("Align Horizontal &Center");

```

Comment:
```suggestion
    sMenuText = QT_TR_NOOP("Horizontal &Center Align");
```
Pending Discord discussion